src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/MenuBarItem.xaml

```diff
@@ -11,7 +11,7 @@
         <Setter Property="BorderThickness" Value="{ThemeResource MenuBarItemBorderThickness}"/>
         <Setter Property="BorderBrush" Value="{ThemeResource MenuBarItemBorderBrush}"/>
         <Setter Property="Margin" Value="{ThemeResource MenuBarItemMargin}"/>
-        <Setter Property="contract7Present:CornerRadius" Value="{ThemeResource ControlCornerRadius}"/>
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}"/>
```

Comment:
I think it's not worth fixing xaml generator to support conditional xaml inside Setter.Property. So this is a little workaround so that xaml generator can find CornerRadius property correctly.